### PR TITLE
Adds multi input wrapper capabilities and example

### PR DESCRIPTION
**Pull request by marina-mosti**
Allows for components that wrap multiple inputs to be able to communicate changes to the schema form through @update-batch

### CODE REVIEW

**Comment by marina-mosti** [comment]
Added some changes also to the documentation as a starting point :D

File: src/form-elements/MultiElementWrapper.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      form: {
+        color: '',
+        song: ''
+      }
+    }
+  },
+  methods: {
+    textInput (field, e) {
+      this.$set(this.form, field, e)
+      this.$emit('update-batch', this.form)
```
Comment (shelru) [inline src/form-elements/MultiElementWrapper.vue]: ok

**Review by afwn90cj93201nixr2e1re — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -6,27 +6,160 @@ features:
...
+```js
+<template>
+    <SchemaForm :schema="mySchema" />
+</template>
+```
+
+The `SchemaForm` will `$emit` **input** events when your components update. This means that you are able to either use `v-model` on it, or manually capture the `@input` event with a method of your own, as well as inject the `:value` property. Either or.
+
+## Component Requirements and the FormMixin
+In order for FormVueLatte's `SchemaForm` to understand your components, they need to implement the `FormMixin.js` that FormVueLatte provides. In its simplest form, make sure that when your `form element` updates, you call the `update` function with the new value.
```
Comment (shentao) [inline docs/README.md:52]: > In its simplest form, make sure that when your `form element` updates, you call the `update` function with the new value.

I think I would rephrase it to something along the lines:

> In it’s simplest form, make sure that when your `form element` wants to make a change to the `value`, it calls the `update` method with the new value.

**Review by shentao — APPROVED** [review]
All good! Just one small comment to the docs. Great work 🚀

File: docs/README.md (hunk 2 above)
Comment (marina-mosti) [reply docs/README.md:52]: Done and pushed
